### PR TITLE
[MI-178] Update Readme to let users know that v2 is now merged and in beta

### PR DESCRIPTION
**Pull request by miogalang**
Update Readme to let users know that v2 is now merged and in beta

/cc @jwswj @joseconsador @mmolina @atroche

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-178

### Risks
* None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,18 @@
 
 [![Build Status](https://travis-ci.org/zendesk/zendesk_api_client_php.svg?branch=master)](https://travis-ci.org/zendesk/zendesk_api_client_php)
 
+## API Client Version
+
+API client version 2 is now in public beta and will be the preferred version of the client to use moving forward. 
+Version 2 will now reside in the master branch and will be supported. 
+
+Version 1 of the API Client will still be available to people using it, it is moved over to the branch `v1` and will still be supported until a stable Version 2 is released.
```
Comment (jwswj) [inline README.md]: Version 1 of the API Client is still available on the [v1](https://github.com/zendesk/zendesk_api_client_php/tree/v1) branch, however will no longer be actively maintained.

File: README.md [hunk 2]
```diff
@@ -2,11 +2,18 @@
 
 [![Build Status](https://travis-ci.org/zendesk/zendesk_api_client_php.svg?branch=master)](https://travis-ci.org/zendesk/zendesk_api_client_php)
 
+## API Client Version
+
+API client version 2 is now in public beta and will be the preferred version of the client to use moving forward.
```
Comment (jwswj) [inline README.md]: Let's be more forward with our wording here: 
This is the second version of the API client which was a complete re-write. The previous version of the API client can be found on the [v1](https://github.com/zendesk/zendesk_api_client_php/tree/v1) branch.

**Comment by miogalang** [comment]
:haircut: @jwswj

**Comment by jwswj** [comment]
Agh - I updated my comment. I've pushed a commit with the changes.

**Comment by jwswj** [comment]
:+1:

**Comment by miogalang** [comment]
:+1: jason's commit

**Comment by joseconsador** [comment]
:+1:

**Pull request merged**